axelor-base/src/main/java/com/axelor/apps/base/service/AddressExportServiceImpl.java

```diff
@@ -60,4 +62,29 @@ public int export(String path) throws IOException {
 
     return addresses.size();
   }
+
+  @Override
+  public Address getAddress(
+      String room,
+      String floor,
+      String streetName,
+      String postBox,
+      String zip,
+      City city,
+      Country country) {
+
+    return addressRepository
+        .all()
+        .filter(
+            "self.room = ?1 AND self.floor = ?2 AND self.streetName = ?3 "
+                + "AND self.postBox = ?4 AND self.zip = ?5, self.city = ?6 AND self.country = ?7",
+            room,
+            floor,
+            streetName,
+            postBox,
+            zip,
+            city,
+            country)
+        .fetchOne();
```

Comment:
Got the following exception when I tried to convert a lead : 
`java.lang.IllegalArgumentException: org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: , near line 1, column 218 [SELECT self FROM com.axelor.apps.base.db.Address self LEFT JOIN self.city _city LEFT JOIN self.country _country WHERE self.room = ?1 AND self.floor = ?2 AND self.streetName = ?3 AND self.postBox = ?4 AND self.zip = ?5, _city = ?6 AND _country = ?7]
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:138)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:181)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:188)
	at org.hibernate.internal.AbstractSharedSessionContract.createQuery(AbstractSharedSessionContract.java:757)
	at org.hibernate.internal.AbstractSharedSessionContract.createQuery(AbstractSharedSessionContract.java:848)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:23)
	at com.axelor.db.Query.fetchQuery(Query.java:390)
	at com.axelor.db.Query.fetch(Query.java:386)
	at com.axelor.db.Query.fetchOne(Query.java:422)
	at com.axelor.db.Query.fetchOne(Query.java:412)
	at com.axelor.apps.base.service.AddressExportServiceImpl.getAddress(AddressExportServiceImpl.java:88)
	at com.axelor.apps.crm.service.ConvertLeadWizardServiceImpl.createPrimaryAddress(ConvertLeadWizardServiceImpl.java:179)
	at com.axelor.apps.crm.web.ConvertLeadWizardController.generateAddress(ConvertLeadWizardController.java:318)
	at com.axelor.apps.crm.web.ConvertLeadWizardController.setFieldsForConversionFromLeads(ConvertLeadWizardController.java:308)`

It's because there is a ',' in the query after "AND self.postBox = ?4 AND self.zip = ?5"